src/api/namespaces/helpers/handlers/collections.ts

```diff
@@ -1,24 +1,37 @@
 import {filterQueryArgs, RequestValues} from '../../utils';
 import {NeftyMarketContext} from '../index';
 import QueryBuilder from '../../../builder';
+import { ApiError } from '../../../error';
 
 export async function getCollectionsAction(params: RequestValues, ctx: NeftyMarketContext): Promise<any> {
     const args = filterQueryArgs(params, {
-        list: {type: 'string', values: ['whitelist', 'blacklist', 'verified', 'nsfw', 'scam']},
+        lists: {type: 'string' },
         sort: {type: 'string', values: ['collection_name', 'list'], default: 'collection_name'},
         order: {type: 'string', values: ['asc', 'desc'], default: 'asc'},
     });
 
-
+    // @TODO: index (collection_name, list)
     const query = new QueryBuilder(`
-                SELECT collection_name, contract, list
```

Comment:
if the contract will be removed from the return value then also update the example JSON
![Screenshot 2022-02-14 at 19 41 39](https://user-images.githubusercontent.com/15836315/153926208-5a90c3de-bdee-48a0-9cf6-dbf8d28f2db9.png)